test/CAS/mccas.swift

```diff
@@ -0,0 +1,34 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -frontend -c -primary-file %s -target arm64-apple-darwin23.0.0 -g -cas-backend -cas-backend-mode=verify -cas-path %t/cas -o %t/test.o
+// RUN: %llvm-dwarfdump %t/test.o | FileCheck %s --check-prefix=VERIFY-FILE
+// VERIFY-FILE: .debug_info
+
+// RUN: %target-swift-frontend -frontend -c -primary-file %s -target arm64-apple-darwin23.0.0 -g -cas-backend -cas-backend-mode=native -cas-path %t/cas -o %t/test.o
+// RUN: %llvm-dwarfdump %t/test.o | FileCheck %s --check-prefix=NATIVE-FILE
+// NATIVE-FILE: .debug_info
+
+// RUN: %target-swift-frontend -frontend -c -primary-file %s -target arm64-apple-darwin23.0.0 -g -cas-backend -cas-backend-mode=casid -cas-path %t/cas -o %t/test.id
+// RUN: cat %t/test.id | FileCheck %s --check-prefix=CASID-FILE
+// CASID-FILE: CASID:Jllvmcas://{{.*}}
+
+// RUN: %target-swift-frontend -frontend -c -primary-file %s -target arm64-apple-darwin23.0.0 -g -cas-backend -cas-emit-casid-file -cas-backend-mode=verify -cas-path %t/cas -o %t/test.o
+// RUN: cat %t/test.o.casid | FileCheck --check-prefix=VERIFY-EMIT
+// VERIFY-EMIT: CASID:Jllvmcas://{{.*}}
+
+// RUN: %target-swift-frontend -frontend -c -primary-file %s -target arm64-apple-darwin23.0.0 -g -cas-backend -cas-emit-casid-file -cas-backend-mode=native -cas-path %t/cas -o %t/test.o
+// RUN: cat %t/test.o.casid | FileCheck --check-prefix=NATIVE-EMIT
+// NATIVE-EMIT: CASID:Jllvmcas://{{.*}}
+
+// RUN: %target-swift-frontend -frontend -c -primary-file %s -target arm64-apple-darwin23.0.0 -g -cas-backend -cas-emit-casid-file -cas-backend-mode=casid -cas-path %t/cas -o %t/test.o
+// RUN: not cat %t/test.o.casid
+
+// RUN: %target-swift-frontend -frontend -c -primary-file %s -target arm64-apple-darwin23.0.0 -g -cas-backend -cas-emit-casid-file -cas-backend-mode=verify -cas-path %t/cas -o -
+// RUN: not cat %t/*.casid
```

Comment:
Please do not use `*` in the `RUN` command. This is not doing what you expect as this `cat` should absolutely succeed in this test case because all you previous `casid` file is in the same directory.